src/client/Microsoft.Identity.Client/Platforms/Android/Broker/AndroidBrokerHelper.cs

```diff
@@ -241,50 +152,25 @@ public void CheckForBrokerAccountInfoInAccountManager(BrokerRequest brokerReques
             throw new MsalUiRequiredException(MsalError.NoAndroidBrokerAccountFound, MsalErrorMessage.NoAndroidBrokerAccountFound);
         }
 
-        private string GetBrokerAccounts(BrokerRequest brokerRequest, Activity callerActivity)
-        {
-            Bundle getAccountsBundle = CreateBrokerAccountBundle(brokerRequest);
-            getAccountsBundle.PutString(BrokerConstants.BrokerAccountManagerOperationKey, BrokerConstants.GetAccounts);
-
-            //This operation will acquire all of the accounts in the account manager for the given client ID
-            var result = _androidAccountManager.AddAccount(BrokerConstants.BrokerAccountType,
-                BrokerConstants.AuthtokenType,
-                null,
-                getAccountsBundle,
-                null,
-                null,
-                GetPreferredLooper(callerActivity));
-
-            Bundle bundleResult = (Bundle)result?.Result;
-            return bundleResult?.GetString(BrokerConstants.BrokerAccounts);
-        }
-
         /// <summary>
         /// This method will acquire all of the accounts in the account manager that have an access token for the given client ID.
         /// </summary>
-        public IReadOnlyList<IAccount> GetBrokerAccountsInAccountManager(BrokerRequest brokerRequest)
+        public IReadOnlyList<IAccount> ExtractBrokerAccountsFromAccountData(string accountData)
         {
-            var accounts = GetBrokerAccounts(brokerRequest, null);
-
-            if (string.IsNullOrEmpty(accounts))
-            {
-                _logger.Info("Android account manager didn't return any accounts. ");
-            }
-
             List<IAccount> brokerAccounts = new List<IAccount>();
 
-            if (!string.IsNullOrEmpty(accounts))
+            if (!string.IsNullOrEmpty(accountData))
             {
-                dynamic authResult = JArray.Parse(accounts);
+                dynamic authResult = JArray.Parse(accountData);
```

Comment:
>uthResult [](start = 25, length = 9)

nit: is there a better name for this? we use authResult a lot, and i don't think this is an authResult as we typically use it.